source/slang/hlsl.meta.slang

```diff
@@ -3116,6 +3120,7 @@ ${{{{
         }
     }
 
+    [ForceInline]
```

Comment:
There are more Interlocked functions that also need to be added. Like in line 3124, 3139, 3155, etc.